### PR TITLE
VIIRS colormap

### PR DESCRIPTION
**Pull request by pholleway**
## Description
Adding file for viirs colormaps. 

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

Please delete options that are not relevant.

- [x] I have performed a self-review
- [] Unit tests pass locally (./scripts/test)
- [x] Code is linted and styled (./scripts/format)

### CODE REVIEW

**Comment by pholleway** [comment]
cc @pjhartzell

**Review by mmcfarland — APPROVED** [review]

File: pctiler/pctiler/colormaps/viirs.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+
+from rio_tiler.types import ColorMapType
+
+viirs_colormaps: Dict[str, ColorMapType] = {
```
Comment (mmcfarland) [inline pctiler/pctiler/colormaps/viirs.py:5]: This needs to be imported into the module `__init__` to get it registered in the service. I think we need to improve how this happens, but for now it's just a manual step.

https://github.com/microsoft/planetary-computer-apis/blob/93ee00039b7da8646b6f13b2b523b9b6acdaad2a/pctiler/pctiler/colormaps/__init__.py#L24-L33

Comment (pholleway) [reply pctiler/pctiler/colormaps/viirs.py:5]: Thanks for the feedback, Matt! viirs_colormaps is import into __init__.py. I'll make a note of this step on our wiki.

**Review by mmcfarland — APPROVED** [review]
Looks good!

**Pull request merged**